include/ginkgo/core/base/lin_op_helpers.hpp

```diff
@@ -0,0 +1,351 @@
+/*******************************<GINKGO LICENSE>******************************
```

Comment:
It does not seem like this file belongs in this PR.

Reply:
Yes, I needed to do this to reduce duplication for the batch linop.